content/en/docs/reference/scheduling/config.md

```diff
@@ -1,30 +1,57 @@
 ---
-title: Scheduling Profiles
+title: Scheduler Configuration
 content_type: concept
 weight: 20
 ---
 
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+The `KubeSchedulerConfiguration` is a configuration API for `kube-scheduler`
```

Comment:
```suggestion
You can customize the behavior of the `kube-scheduler` by writing a configuration
file and passing its path as a command line argument.
```

(the name KubeSchedulerConfiguration isn't a Kubernetes resource, and mentioning it might confuse some readers.

Reply:
Done